src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.java

```diff
@@ -438,6 +438,14 @@ public void getFullOffenderInformation_byOffenderNo() {
         assertThatJsonFileAndStatus(response, 200, "offender_extra_info.json");
     }
 
+    @Test
+    public void getFullOffenderInformation_byOffenderNoAlt() {
```

Comment:
what does `noAlt` refer to in `getFullOffenderInformation_byOffenderNoAlt()` ?

Reply:
Alt was alternative!  Sorry horrible wording